content/docs/guides/responsive_amp/responsive_design@pt_BR.md

```diff
@@ -0,0 +1,173 @@
+---
+$title: Criar páginas AMP responsivas
+---
+[TOC]
+
+## Introdução
+
+O Web design responsivo é a criação de páginas da Web simples de usar e que atendem às necessidades dos usuários. Essas páginas se ajustam ao tamanho e à orientação da tela do dispositivo.  Com AMP, é muito fácil conseguir isso.  Ela é compatível com todas as categorias de tela e dispositivos e oferece componentes responsivos integrados.
+
+Neste guia, você verá como implementar facilmente estes princípios básicos em páginas AMP:
+
+- [Como controlar a janela de visualização](#como-controlar-a-janela-de-visualização)
+- [Como criar um layout responsivo](#como-criar-um-layout-responsivo)
+- [Como dimensionar mídias](#como-dimensionar-mídias-na-página)
+
+## Como controlar a janela de visualização
+
+Para otimizar sua página da Web e fazer com que o conteúdo seja dimensionado e se ajuste ao tamanho da janela do navegador em qualquer dispositivo, é preciso especificar o elemento `meta` da janela de visualização. Esse elemento indica ao navegador como dimensionar e ajustar a área visível (a janela de visualização) da página da Web.
+
+Quais valores devem ser usados?  Isso já está especificado nas páginas AMP.  Como parte da [marcação obrigatória](/pt_br/docs/reference/spec.html#required-markup) das páginas AMP, especifique a janela de visualização a seguir:
+
+```html
+<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+```
+
+Essas são as configurações comuns da janela de visualização que você usaria para um site responsivo. Embora `initial-scale=1` não seja obrigatório para uma página AMP válida, esse elemento é recomendado porque define o nível de zoom como 1 quando a página é carregada pela primeira vez.
+
+## Como criar um layout responsivo
+
+No design responsivo, você pode usar consultas de CSS [`@media`](https://developer.mozilla.org/docs/Web/CSS/@media) para definir o estilo da sua página da Web em várias dimensões de tela. É possível fazer isso sem alterar o conteúdo da página.  Em uma página AMP, você pode continuar usando essas mesmas consultas de CSS `@media`. Além disso, para um controle mais preciso sobre o elemento da página AMP, especifique nele o atributo `media`. Isso será muito útil caso você precise mostrar ou ocultar um elemento com base em uma consulta de mídia. Consulte a seção [Como alterar a direção de arte de uma imagem](#changing-the-art-direction-of-an-image) para ver um exemplo que usa o atributo `media`.
```

Comment:
Translate `#changing-the-art-direction-of-an-image` -> Should correspond to header within this doc.